msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,306 @@
+        | Client id | Saved previously as your **Microsoft App ID**. |
+        | Client secret | Saved previously as **Value** of the client secret ID. |
+        | Token Exchange URL | Use the **Application ID URL** obtained previously while exposing API endpoint. |
+        | Tenant ID | Enter **common**. |
+        | Scopes | Enter **User.Read** and also add all the **Scopes** required while specifying permissions to downstream APIs. |
+
+        :::image type="content" source="./assets/images/adaptive-cards/oauth-connection-setting.png" alt-text="Bot service connection":::
+
+    1. Select **Save**.
+
+- title: Test your app
+  durationInMinutes: 2
+  content: |
+
+    The most comprehensive way to test your app is by creating an app package and uploading it to Teams. This method is the only way to test the full functionality available to your app,
+    across all scopes. You can configure and run the Teams app by uploading the manifest.zip file.
+
+    To run the Teams file bot you need to upload the manifest.zip file in Teams.
+
+    1. In the File Explorer, go to botbuilder-samples > samples > csharp_dotnetcore > 56.teams-file-upload > appsettings.
+
+    2. Open the appsettings in visual studio and update the appsettings.json configuration for the bot to use the `MicrosoftAppId`, `MicrosoftAppPassword`, `MicrosoftAppType`, `MicrosoftAppTenantID`
+
+       ![app settings](~/assets/images/app-settings.png)
+    
+    3. Open the manifest.json file in visual studio.
+
+    4. In the manifest.json file, find YOUR-MICROSOFT-APP-ID and replace it with your App Id (that was created when you registered your bot earlier) every where you see the place holder string and save it.
+       
+       ![manifest json](~/assets/images/manifest-json.png)
+
+    5. Zip up the contents of the `TeamsAppManifest' folder to create **manifest.zip**. Ensure to open the manifest file in visual studio code and replace with your MicrosoftAppID (depending on the scenario the Microsoft App Id may occur multiple times in the manifest.json)
+
+    6. Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
+
+    7. To upload **manifest.zip**, select **Store > Manage your apps > Upload a custom app** and open **manifest.zip**.
+
+    8. Select **Add** to add the bot to your chat.
+
+    9. Before sending a message to the bot, go to File Explorer, navigate to botbuilder-samples > samples > csharp_dotnetcore > 56.teams-file-upload 
+
+    6. Open TeamsFileUpload.csproj in visual studio and ensure you are in 'Teams file upload'
+
+      ![Visual code](~/assets/images/visual-studio-teams-file-upload-bot.png)
+
+    4. Run your bot, from Visual Studio with `F5` or using `dotnet run` in the appropriate folder
+
+    8. Send a message `Hi` to the bot and wait for few seconds for it to respond with a card.
+
+- title: Interacting with the bot in Teams
+  durationInMinutes: 2
+  content: |
+
```

Comment:
Add a introduction to this page. Also there is a note missing from the readme file.